ui/src/graphics.tsx

```diff
@@ -169,18 +182,27 @@ export const
       },
       render = () => {
         const
-          { view_box, width, height, stage, scene } = state,
+          { view_box, width, height, stage, scene, type, image, path } = state,
           stageEls = stage ? unpack<Recs>(stage).map(renderEl) : [],
           sceneEls = scene ? unpack<El[]>(scene).map(({ d, o }, i) => renderEl({
             ...(d ? JSON.parse(d) : {}),
             ...(o ? JSON.parse(o) : {}),
-          }, i)) : []
+          }, i)) : [],
+          backgroundImageSrc = path 
+            ? 'url('+path+')' 
+            : image ? `url(data:image/${type};base64,${image})` 
+            : undefined 
+            
         return (
-          <div data-test={name} className={css.card}>
-            <svg viewBox={view_box} width={width} height={height}>
-              <g>{stageEls}</g>
-              <g>{sceneEls}</g>
-            </svg>
+          <div style={{ backgroundImage: `${backgroundImageSrc ? backgroundImageSrc : undefined}`, 
```

Comment:
Is this wrapper necessary? Wouldn't just adding `style` to an existing wrapper be enough?

Reply:
@mturoci, that was my original idea too. But the problem is that the existing `div` is rendered with an absolute positioning offset of 15px around all sides. So when I put the background-image on that `div`, it does not cover the entire card. Here's a screen show showing what that looks like:

![issue1817_backgroundBorder](https://github.com/user-attachments/assets/a47606f4-7497-4e06-b1ca-baa5dd061bd2)

I tried various things to adjust the styling of the `div` --  I tried setting `margin` and `padding`, but nothing looked right. Plus I was concerned about backwards compatibility issues when changing the styling of the main `div` with the graphics. As a result, I decided to add a new parent `div`. If you can think of another approach I'm loverlooking, let me know!

Reply:
I see. I think the offset is fine though because even though the image in your current implementation would take up the whole card, the outer 15px wouldn't be reachable by graphics render, if I understood it correctly, which would be a problem especially when a user would want to highlight a certain part of the image with a rectangle for example.

Reply:
Thanks @mturoci. So just to be clear, you are saying to remove the new parent div and instead put the background image on the existing div, which is defined with the 15px offset - right? This means the background image will not cover the entire card, as you can see in this screen shot. Please confirm that I'm understanding correctly. Thanks!!

![issue1817_withBorders](https://github.com/user-attachments/assets/6b22cabb-978e-485c-8129-bbfd27c20609)

Reply:
Correct :)

Reply:
@mturoci I made the changes as suggested. Removed the parent div and added the background-image stuff to the existing div. Here's a screen shot of the custom example I've been using:

![issue1817_afterChanges](https://github.com/user-attachments/assets/89f15c4d-4353-49c2-be14-8fc1a01d0c7e)